packages/palette/src/utils/usePosition.ts

```diff
@@ -44,7 +45,7 @@ export const usePosition = ({
   const tooltipRef = useRef<HTMLElement | null>(null)
   const anchorRef = useRef<HTMLElement | null>(null)
 
-  useLayoutEffect(() => {
+  useIsomorphicLayoutEffect(() => {
```

Comment:
Or maybe just use `useEffect`? Is there a noticeable difference between the two UX-wise?

Reply:
Oh, yeah I'm intentionally using useLayoutEffect here. It's kind of wild actually; the return from `getBoundingClientRect` is incorrect for long enough for the position to be wrong. You see it show up as a very brief flash as the popover repositions itself.

Reply:
Cool 👍 Figured as much but wanted to be sure